Source/Core/DolphinQt/TAS/GBATASInputWindow.cpp

```diff
@@ -27,25 +27,23 @@ GBATASInputWindow::GBATASInputWindow(QWidget* parent, int controller_id)
   setWindowTitle(tr("GBA TAS Input %1").arg(controller_id + 1));
 
   m_b_button =
-      CreateButton(QStringLiteral("&B"), GBAPad::BUTTONS_GROUP, GBAPad::B_BUTTON, &m_overrider);
+      CreateButton(QStringLiteral("&B"));
```

Comment:
This doesn't have anything to do with my overhaul from what I can tell. Are we sure this should be modified?

Reply:
Also if you DO make this change, these lines are short enough such that it should all be on the same line

Reply:
So, both these changes I removed from Dolphin upstream mainly because your overhaul does use some of the older methods, so I had to revert in order for it to work. I didn't have much time to do the opposite, aka changing your overhaul to use the new methods. Sorry for the confusion. I'm also outside so it's not too easy for me to look into this right now :/